desktop/core/src/desktop/api2.py

```diff
@@ -128,6 +129,10 @@ def get_config(request):
   config['hue_config']['enable_chunked_file_uploader'] = ENABLE_CHUNKED_FILE_UPLOADER.get()
   config['hue_config']['enable_task_server'] = TASK_SERVER_V2.ENABLED.get()
   config['hue_config']['restrict_file_extensions'] = RESTRICT_FILE_EXTENSIONS.get()
+  config['hue_config']['concurrent_max_connection'] = CONCURRENT_MAX_CONNECTIONS.get()
+  config['hue_config']['file_upload_chunk_size'] = FILE_UPLOAD_CHUNK_SIZE.get()
+  config['hue_config']['enable_file_download_button'] = SHOW_DOWNLOAD_BUTTON.get()
```

Comment:
These configs are already present in `desktop/core/src/desktop/templates/global_js_constants.mako`

Reply:
Nice catch, removed it from global file.

Reply:
Make sense, I have moved it. Thanks